var/spack/repos/builtin/packages/llvm/package.py

```diff
@@ -87,6 +87,7 @@ class Llvm(CMakePackage):
     depends_on('ncurses', when='+lldb')
     depends_on('swig', when='+lldb')
     depends_on('libedit', when='+lldb')
+    depends_on('py-six', when='+lldb +python')
```

Comment:
but also only for `llvm@5:` right?

Reply:
@healther yes, thanks for catching that.